### ISSUE TITLE
Hifi + Parental date /

### ISSUE THREAD

**Issue by cbirbes** [chhylp123/hifiasm#281]

**Comment by chhylp123** [comment]
What is the hamming error rate of trio-binning assemblies? Looks like the hamming error rate is very high. The unbalanced size issues are often caused by wrong hom/het peaks (see: https://hifiasm.readthedocs.io/en/latest/faq.html#how-can-i-tweak-parameters-to-improve-hi-c-integrated-assembly).

**Comment by cbirbes** [comment]
These results are from trio assembly.

I don't have HiC for my bovine genome but on goat the HiC assembly seems better than trio (in term of haplotype size):
hap1 HiC :
2.913.416.693 bp
W       103733  3449742 0.030070
H       417700  3449938 0.121075

hap2 HiC:
2.764.934.144 bp
W       202348  3798587 0.053269
H       987714  3798753 0.260010

hap1 Trio:
3.090.251.445 bp
W       75148   3839676 0.019571
H       102934  3840084 0.026805

hap2 Trio:
2.576.405.207 bp
W       227666  3385745 0.067243
H       1133417 3386144 0.334722

the Hamming error rate of both Hap2 is quite high no ?

**Comment by chhylp123** [comment]
Yes, very high. Probably you should check if the parental data is correct.
